app/main/posts/modify/post-tabs.directive.js

```diff
@@ -48,11 +48,10 @@ function PostVerticalTabsController(
     }
 
     function stageIsComplete(stageId) {
-        return _.includes($scope.post.completed_stages, stageId);
+        return $scope.post.completed_stages.indexOf(stageId) > -1;
```

Comment:
This wasn't intended to fix the bug itself, this fixed a secondary bug that appeared once the directive scope loaded. The stageIsComplete function was caught in a loop.